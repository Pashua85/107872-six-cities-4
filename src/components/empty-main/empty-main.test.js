import React from 'react';
import Enzyme, {mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import EmptyMain from './empty-main';

Enzyme.configure({
  adapter: new Adapter()
});

describe(`EmptyMain`, () => {
  it(`should render correctly`, () => {
    const wrapper = mount(<EmptyMain />);
    expect(wrapper).toMatchSnapshot();
  });
});
