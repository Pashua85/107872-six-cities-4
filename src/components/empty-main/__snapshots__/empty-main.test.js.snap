// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyMain should render correctly 1`] = `ReactWrapper {}`;
